src/lib/rag.ts

```diff
@@ -7,40 +7,40 @@ import { Runnable } from "@langchain/core/runnables";
 import { queryStore } from "./api";
 
 export class RagEngine {
-    qaChain: Runnable;
-    queryEnhancerChain: Runnable;
+    // qaChain: Runnable;
+    // queryEnhancerChain: Runnable;
 
     constructor() {
-        const model = new ChatOpenAI({
-            configuration: {
-                baseURL: logseq.settings!["OPENAI_BASE_URL"] as string,
-                apiKey: logseq.settings!["OPENAI_API_KEY"] as string,
```

Comment:
The `!` freeze the app on when you first open `mod+p` if you have not yet set your api key. I just comment out this unused code for now.